elyra/pipeline/processor_kfp.py

```diff
@@ -174,7 +174,7 @@ def _cc_pipeline(self, pipeline, pipeline_name):
                                      cos_directory=cos_directory,
                                      cos_pull_archive=operation_artifact_archive,
                                      pipeline_outputs=self._artifact_list_to_str(operation.outputs),
-                                     pipeline_inputs=self._artifact_list_to_str(operation.inputs),
+                                     pipeline_inputs=self._artifact_list_to_str(operation.file_dependencies),
```

Comment:
Based on my understanding of this from our discussion, this is what you actually want to fix this:
```suggestion
                                     pipeline_inputs=self._artifact_list_to_str(operation.inputs + operation.file_dependencies),
```